variable/technology/technologies.yaml

```
@@ -1089,3 +1089,10 @@ Rate Automatic Frequency Restoration Reserve|Electricity|Wind|OffShore:
    description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
                 for a Wind OffShore power plant
    unit: '%'
+   
+# description of Demand flexibility potentials
+   
+Load Shifting Potential|Electricity|Residential|AC:
+   description: Maximum shiftable load in a given region/time frame,
+                due to households changing their air conditioning usage.
+   unit: 'MW'
```

No need to put strings in parentheses... (only special characters like %)
```suggestion
   unit: MW
```